.github/workflows/pytest_slow.yml

```diff
@@ -68,13 +71,14 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.LUDWIG_TESTS_AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.LUDWIG_TESTS_AWS_SECRET_ACCESS_KEY }}
       EXCLUDED_MARKERS: "benchmark"
+      TOKENIZERS_PARALLELISM: false
 
     name: py${{ matrix.python-version  }}, torch-${{ matrix.pytorch-version }}, ${{ matrix.test-markers }}, ${{ matrix.os }}, ray ${{ matrix.ray-version }}
     if: needs.start-runner.result != 'skipped'
     needs: start-runner # required to start the main job when the runner is ready
     runs-on: ${{ needs.start-runner.outputs.label }} # run the job on the newly created runners
 
-    timeout-minutes: 90
+    timeout-minutes: 120
```

Comment:
does it actually need this long?

Reply:
No, I was just debugging. Will update once the current job finishes running